perfmetrics/scripts/vm_metrics/vm_metrics.py

```diff
@@ -1,92 +1,118 @@
-"""
-To execute the script:
->>python3 vm_metrics.py {instance} {start time in epoch sec} {end time in epoch sec} {mean period in sec}
-
-The code takes input the start time and end time (in epoch seconds) and the
-instance name and the mean period. Then it creates an instance of VmMetrics
-class and calls its methods with all the corresponding parameters, makes the api
-call, reads the response and returns a list of metric points where each point
-has the peak value and mean value of a mean period. Then it dumps all the data
-into a google sheet.
+"""Extracts required Google Cloud metrics using Monitoring V3 API call, parses
+   the API response into a list and writes to google sheet.
+
+   Takes VM instance name, interval start time, interval end time, alignment
+   period, fio test type and google sheet name as command line inputs.
+   The supported fio test types are: read, write
+   Metrics extracted:
+   1.Peak Cpu Utilization(%)
+   2.Mean Cpu Utilization(%)
+   3.Peak Network Bandwidth(By/s)
```

Comment:
by is not the short form for bytes. Please dont use it anywhere. rename everywhere to bytes